doc/tutorials/application-integration.md

```diff
@@ -185,12 +180,12 @@ Thus we are interested in the value we use:
 char *val = keyString(k);
 ```
 
-Finally, we need to convert the configuration value to the datatype we
+Now we need to convert the configuration value to the data-type we
```

Comment:
strange recommendations: "now" is not a good style, "finally" is not necessary but okay.

data-type also seems very non standard: it seems to be "data type" see #920